pkg/whitesource/whitesource.go

```diff
@@ -35,6 +36,7 @@ type Request struct {
 	RequestType  string `json:"requestType,omitempty"`
 	UserKey      string `json:"userKey,omitempty"`
 	ProductToken string `json:"productToken,omitempty"`
+	ProductName  string `json:"productName,omitempty"`
```

Comment:
If we support `productName` I think we should also support `projkectName`, @nevskrem?